### PR TITLE
Fix: Show media preview in Safari 13 and other browsers that don't support `aspect-ratio` [ED-9847]

### PR DESCRIPTION
**Pull request by rami-elementor**
Fix https://github.com/elementor/elementor/issues/21368

### CODE REVIEW

File: assets/dev/scss/editor/panel/controls/_media.scss [hunk 1]
```diff
@@ -36,6 +36,7 @@
 
 	&__preview {
 		height: 100%;
+		min-height: 100px; // Fix for Safari 13 and other browsers that don't support aspect-ratio
```
Comment (StyleShit) [inline assets/dev/scss/editor/panel/controls/_media.scss:39]: why don't you just revert to the original padding hack?

Comment (rami-elementor) [reply assets/dev/scss/editor/panel/controls/_media.scss]: Done. aspect ration 21:9 like we currently have in modern browsers.

**Review by StyleShit — DISMISSED** [review]

File: assets/dev/scss/editor/panel/controls/_media.scss [hunk 2]
```diff
@@ -36,6 +36,7 @@
 
 	&__preview {
 		height: 100%;
+		padding-bottom: 42.8571%; // BC: Fix for older browsers that don't support "aspect-ratio: 21 / 9;"
```
Comment (arielk) [inline assets/dev/scss/editor/panel/controls/_media.scss:39]: @rami-elementor please add TODO: Remove this line in after we will deprecate Safari v13

Comment (rami-elementor) [reply assets/dev/scss/editor/panel/controls/_media.scss]: Done.

**Review by arielk — APPROVED** [review]

**Pull request merged**